.github/ISSUE_TEMPLATE/user-problem.md

```diff
@@ -18,7 +18,13 @@ Put the problem summary here.
 Diagnostic files
 ----------------
 
-If a crash (gpf, assert, abort, etc) or OutOfMemory condition, please provide the diagnostic files produced (javacore, Snap, jitdump, core). The smaller files can be attached to this Issue. The core should be compressed and made available via a file sharing service (Box, Google Drive, etc). If there are privacy concerns please direct email the files to an OpenJ9 committer.
+If a crash (gpf, assert, abort, etc) or OutOfMemory condition, please provide the diagnostic files produced (core, javacore, jitdump, Snap). The smaller files can be attached to this Issue.
```

Comment:
This is missing something: perhaps it should begin "If you experience a crash..."?
"GPF" should be capitalized, and "etc" should be "etc.".